Doc/whatsnew/3.8.rst

```diff
@@ -1685,7 +1842,7 @@ Changes in the Python API
 * The ``PyGC_Head`` struct has changed completely.  All code that touched the
   struct member should be rewritten.  (See :issue:`33597`.)
 
-* The ``PyInterpreterState`` struct has been moved into the "internal"
+* The :c:type:`PyInterpreterState` struct has been moved into the "internal"
   header files (specifically Include/internal/pycore_pystate.h).  An
```

Comment:
```suggestion
  header files (specifically :source:`Include/internal/pycore_pystate.h`).  An
```
There's a Sphinx role to properly link to the source code, `:source:`. Do you think it might be useful here? This allows users to directly click on it to be referred to the source code of the same version, in this case it would convert to: [Include/internal/pycore_pystate.h](https://github.com/python/cpython/blob/3.8/Include/internal/pycore_pystate.h).

Reply:
That's the job of the main docs.  The whatsnew is an overview.